server/queue/queue-api/src/test/java/org/apache/james/queue/api/MailQueueFixture.java

```diff
@@ -0,0 +1,51 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.queue.api;
+
+import static org.apache.mailet.base.MailAddressFixture.RECIPIENT1;
+import static org.apache.mailet.base.MailAddressFixture.RECIPIENT2;
+import static org.apache.mailet.base.MailAddressFixture.SENDER;
+
+import java.util.Date;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+import org.apache.mailet.base.test.FakeMail;
+import org.apache.mailet.base.test.MimeMessageBuilder;
+
+public interface MailQueueFixture {
```

Comment:
A Fixture is supposed to be about built objects, not util methods.

Reply:
builder is statefull, so having it as a field won't work.

Would you consider cleaner to have yet another separated MailQueueTestUtils class? Me no.

Reply:
Are you arguing that using Fixture word for the wrong use case is not a problem ? Why would we choose this word if it does not talk about util methods ?

Reply:
What wording would you propose?

Reply:
`Mails` ?

Reply:
Sorry I don't understand.

Reply:
I suggest the class to be called `Mails` like for example `Lists` help building List in guava.